pc/manager.hpp

```diff
@@ -221,6 +224,10 @@ namespace pc
     void poll_schedule();
     void reset_status( int );
 
+    // sends pending updates if the previous send was more than PC_FLUSH_INTERVAL ago
+    // or there is >= get_max_batch_size() pending updates
```

Comment:
Could we copy the docstring from the previous `user::send_pending_ups` function?